portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertFeedback.testcase

```diff
@@ -38,16 +38,33 @@ definition {
 		else {
 			JSONLayout.deletePublicLayout(
 				groupName = "Guest",
-				layoutName = "Clay Sample Test Page");
+				layoutName = "Clay Page");
 		}
 	}
 
-	@description = "Check if each nav section of clay sample portlet looks normal from screenshot"
-	@priority = 3
+	@description = "LPS-110903 - Alerts are not styled correctly"
+	@priority = 5
 	test IconCanDisplayTooltip {
+		task ("Given Clay Sample Portlet") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "Clay Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Guest",
+				layoutName = "Clay Page",
+				widgetName = "Clay Sample");
 
-		// TODO LPS-133020 IconCanDisplayTooltip pending implementation
+			Navigator.gotoPage(pageName = "Clay Page");
+		}
```

Comment:
Hey @andreluis2022 could you remove this part? These steps are being covered in the set up

Reply:
Done!